docs/pages/guides/alternatives-for-expo-notifications.mdx

```diff
@@ -0,0 +1,73 @@
+---
+title: Alternatives for Expo Notifications
+description: Learn about alternative services for sending push notifications.
```

Comment:
```suggestion
description: Learn about alternative push notification services for Expo Notifications.
```

Let's add the context int he description that the services listed are in context to Expo Notifications.